test/integration/full/target-size/too-many-rects.js

```diff
@@ -9,6 +9,7 @@ describe('target-size too many rects test', () => {
         elementRef: true
       };
       const context = {
+        include: '#incomplete',
```

Comment:
On chrome for some reason it was failing with 2 incomplete instead of 1 where the other was the target `h1 a` (which is a mocha heading).